### PR TITLE
Update Caption -vvaths

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @vvaths
- [ ] @Bharatsalian01
- [ ] @Krishna-Varada
- [ ] @hsandadi1234
- [ ] @satishpsk7
- [ ] @srajasekar12
- [ ] @sthumu013
- [ ] @tanjain10

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Awesome new caption!
```
Comment (vyas000) [inline _slides/02-vvaths.md:6]:
```suggestion
Awesome Magic caption!!!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/02-vvaths.md:5]: Great job!

**Review by vyas000 — CHANGES_REQUESTED** [review]
Please make changes requested! good job!

**Review by srikanthpothuraju — APPROVED** [review]

File: _slides/02-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/02-vvaths.md:5]: thanks :)

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**